app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -1,64 +1,48 @@
 package com.codesoom.assignment.application;
 
-import com.codesoom.assignment.ProductNotFoundException;
 import com.codesoom.assignment.domain.Product;
-import com.codesoom.assignment.domain.ProductRepository;
 import com.codesoom.assignment.dto.ProductData;
-import org.springframework.stereotype.Service;
 
-import javax.transaction.Transactional;
 import java.util.List;
 
-@Service
-@Transactional
-public class ProductService {
-    private final ProductRepository productRepository;
+/**
+ * 상품 조회, 수정, 삭제 서비스
+ */
+public interface ProductService {
+
+    /**
+     * 상품들을 조회
+     * @return 상품 배열을 리턴합니다.
+     */
+    List<Product> getProducts();
+
+    /**
+     * 상품 상세 조회
+     * @param id 조회할 상품의 id
+     * @return 상품을 리턴합니다.
+     */
```

Comment:
메인 디스크립션과 `@return`을 헷갈려하시는 것 같아용.
```suggestion
    /**
     * 상품 하나를 조회해 리턴합니다.
     * @param id 조회할 상품의 id
     * @return 조회된 상품
     */
```

Reply:
앞으로 헷갈리지 않게 작성할 수 있을거 같습니다 ㅎㅎ!